build.gradle

```diff
@@ -49,7 +49,6 @@ ext {
   gradleVersion = versions.gradle
   minClientJavaVersion = 11
   minNonClientJavaVersion = 17
-  // The connect:api module also belongs to the clients module, but it has already been bumped to JDK 17 as part of KIP-1032.
```

Comment:
`connect-api` does not belong to the clients module, so I just removed this comment.